content/guides/outliers.md

```diff
@@ -77,9 +77,9 @@ To ensure that your alert is properly calibrated, you can set the time window at
 {: #algorithms}
 
 
-There are two different outlier detection algorithms you can use on your data: DBSCAN and Median Absolute Deviation (MAD). We recommend starting with the default algorithm, DBSCAN. If you have trouble detecting the right outliers, you can adjust the parameters to DBSCAN or try the alternate algorithm, MAD. Explanation of each algorithm and its parameters follows.
+There are two different types of outlier detection algorithms you can use on your data: DBSCAN/Scaled DBSCAN and MAD (Median Absolute Deviation)/Scaled MAD. We recommend starting with the default algorithm, DBSCAN. If you have trouble detecting the right outliers, you can adjust the parameters to DBSCAN or try the alternate algorithm, MAD. If your metrics have a larger scale and you are seeing outliers within a closely clustered group, try the scaled algorithms, Scaled DBSCAN or Scaled MAD. Explanation of each algorithm and its parameters follows.
```

Comment:
"If you have metrics on a larger scale that look to be closely clustered but DBSCAN/MAD algorithms are identifying some as outliers, try the scaled algorithms." (I think when this is undesired behavior is implicit)